unstructured/ingest/connector/clarifai.py

```diff
@@ -0,0 +1,129 @@
+import os
+import typing as t
+from dataclasses import dataclass, field
+import uuid
+from pathlib import PurePath
+
+from unstructured.ingest.enhanced_dataclass import enhanced_field
+from unstructured.ingest.enhanced_dataclass.core import _asdict
+from unstructured.ingest.interfaces import (
+    AccessConfig,
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseSingleIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.ingest.error import DestinationConnectionError
+from unstructured.ingest.utils.data_prep import chunk_generator
+from unstructured.staging.base import flatten_dict
+from unstructured.utils import requires_dependencies
+
+if t.TYPE_CHECKING:
+    from clarifai.client.input import Inputs
+
+@dataclass
+class ClarifaiAccessConfig(AccessConfig):
+    api_key: str = enhanced_field(sensitive=True)
+    #print("class initialized")
+
+@dataclass
+class SimpleClarifaiConfig(BaseConnectorConfig):
+    access_config: ClarifaiAccessConfig
+    app_id: t.Optional[str] = None
+    user_id: t.Optional[str] = None
+    
+@dataclass
+class ClarifaiWriteConfig(WriteConfig):
+    batch_size: int = 50
+    num_processes: int = 1
```

Comment:
You can remove this whole num_processes idea from the whole PR (including in the cli/cmds and the .sh). It doesn't look like it is being used. We were trying to use multiprocessing in the destinations (it works in source) for a while, and it worked with Pinecone. But most of the other destinations, their api's couldn't handle it anyway.
So for now we will keep it simple. Remove any of the `num_processes` stuff.